source/_components/device_tracker.tplink.markdown

```diff
@@ -15,6 +15,11 @@ ha_release: pre 0.7
 
 This platform allows you to detect presence by looking at connected devices to a [TP-Link](https://www.tp-link.com) device. This includes the ArcherC9 line.
 
+<p class='note'>
+TP-Link devices typically only allow one login at a time to the admin console.  This component will count torwards your one allowed login. Depending on how aggressively you configure device_trackier you may not be able to access the admin console of your TP-Link device without first stopping Home Assistant.
```

Comment:
"device_tracker" instead of "device_trackier".

You may also want to add a note about stopping Home Assistant and (I assume) waiting a few minutes for the session to expire.